samples/hello-workflow-fragment/src/main/java/com/squareup/sample/helloworkflowfragment/HelloWorkflowFragment.kt

```diff
@@ -1,15 +1,40 @@
 package com.squareup.sample.helloworkflowfragment
 
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.activityViewModels
+import androidx.lifecycle.SavedStateHandle
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
 import com.squareup.workflow1.SimpleLoggingWorkflowInterceptor
-import com.squareup.workflow1.ui.WorkflowFragment
-import com.squareup.workflow1.ui.WorkflowRunner
+import com.squareup.workflow1.ui.WorkflowLayout
 import com.squareup.workflow1.ui.WorkflowUiExperimentalApi
+import com.squareup.workflow1.ui.renderWorkflowIn
+import kotlinx.coroutines.flow.StateFlow
 
 @OptIn(WorkflowUiExperimentalApi::class)
-class HelloWorkflowFragment : WorkflowFragment<Unit, Nothing>() {
-  override fun onCreateWorkflow(): WorkflowRunner.Config<Unit, Nothing> {
-    return WorkflowRunner.Config(
-        HelloWorkflow, interceptors = listOf(SimpleLoggingWorkflowInterceptor())
-    )
+class HelloWorkflowFragment : Fragment() {
+  private val model: HelloViewModel by activityViewModels()
```

Comment:
Confusing that you scoped the ViewModel to the activity and not the fragment here?? From what I can tell in this example we have a single Fragment and not a shared ViewModel. So we should be delegating to `by viewModels()` here and scoping this to the fragment no?

Reply:
Yeah, that was lazy. `by viewModels()` doesn't work for `Fragment` of course, because that would be too useful.

- [ ] Fix scoping in `Fragment` sample